index.html

```diff
@@ -0,0 +1,149 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800&display=swap" rel="stylesheet">
+    <title></title>
+</head>
+<body>
+    <!-- header -->
+    <div class="header">
+        <div class="nav">
+            <div class="nav__rightside">
+                <div><h2 class="nav__logo">Medical Core</h2></div>
+                <div class="nav__items">
+                    <a class="nav__item">home</a>
+                    <a class="nav__item" alt="">product</a>
+                    <a class="nav__item" alt="">pricing</a>
+                    <a class="nav__item" alt="">contact</a>
+                </div>
+            </div>
+            <div class="button">
+                <a class="button__linkgreen">login</a>
+                <button class="button__btn--primary">Join us <i class="bi bi-arrow-right-short"></i></button>
```

Comment:
change this to "button button__btn--primary" . This is an element's modifier and it is  not recommended in BEM to use it as a class alone